pkg/controller/shatag/shatag.go

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright 2020 IBM Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package shatag
+
+import (
+	"os"
+	"strings"
+)
+
+
+
+func GetImageRef(envVar string ) string{
+
+	var imageSuffix string
+	imageTagOrSHA := os.Getenv(envVar)
```

Comment:
Everything looks good. One suggestion would be to check whether env vars exist or not. If not, then default to the tag that you were setting previously through code. You can take a look at this code snippet: 
https://github.com/IBM/ibm-cert-manager-operator/blob/master/pkg/resources/util.go#L37
https://github.com/IBM/ibm-cert-manager-operator/blob/master/pkg/resources/util.go#L51

This is just in case if there are no env vars at all (if someone deleted it), there should be some way to launch operands (i.e. by using the default tags)

Reply:
Swati, there is a discussion going on to change this approach of image tag. So, we will handle this minor change in another PR if needed (that is, if we are not moving to the new method)